src/content/en/2021/page-weight.md

```diff
@@ -1,21 +1,176 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=1911661099&format=interactive",
+  sheets_gid="38001790",
+  sql_file="requests_format_distribution.sql"
+) }}
+
+We know that images responsible for a large percentage of web page weight. The above graphic shows the top sources of image weight and the weight distribution. Top 3: JPG, WebP and PNG. So not only is the JPG the most popular image format, it also tends to be the largest by size as well - even larger than a lossless format like the PNG. Likely, the rise in popularity for WebP is due to all major browsers finally supporting the file format. PNG remains popular for use cases such as icons and logos.
+
+### Image bytes
+
+{{ figure_markup(
+  image="response-distribution-images.png",
+  caption="Distribution of image response sizes per page.",
+  description="Bar chart showing the distribution of image bytes per page. Desktop pages tend to load more image bytes per page throughout the distribution. The 10, 25, 50, 75, and 90th percentiles for mobile pages are: 62, 257, 877, 2,324, and 4,992 KB of images per page.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=1440589973&format=interactive",
+  sheets_gid="895329225",
+  sql_file="request_type_distribution_2021.sql"
+) }}
+
+Looking at total image bytes shows us that this metric has remained virtually unchanged from the previous year. This is likely due more and more images being served by content distribution networks (CDN), which apply strong optimizations to images as they are uploaded to their servers for use on future requests.
+
+## Conclusion
+
+The overall trend moving forward will likely continue to grow for desktop rendered pages and remain mostly unchanged for mobile. Just as they did, along with other browser vendors, for effort to move sites to HTTPS, Google has applied pressure on site owners to put their web pages on a diet, especially for mobile. Google's strategy has been solitified into a program where certain metrics (now called Core Web Vitals) that significantly impact the user experience reward web sites with smaller, faster loading pages by raising their rankings in search results. Conversely, site owners who continue to build heavy pages will be penalized in search results. In a world where small changes in search rankings can have a big impact on the success or failure of a site and their owner's, we should see greater emphasis on creating lighter web pages from design to hosted services that continue to find technological solutions to help keep page assets small and transported in a much faster way.
```

Comment:
The first two sentences are really confusing. Also not sure we should suggest a future trend? The rest of the paragraph could use a lot of slimming.

Reply:
Agreed. It should be more like the latest trends suggest. Will fix the conclusion up.